cmd/api/routers/routers.go

```diff
@@ -108,6 +108,27 @@ func (c *Controller) GetNamespacedResourceByName(context *gin.Context) {
 	context.JSON(http.StatusOK, resource)
 }
 
+func (c *Controller) GetLogsURLByResourceName(context *gin.Context) {
```

Comment:
Just for consistency.

```suggestion
func (c *Controller) GetLogURLsByResourceName(context *gin.Context) {
```

Reply:
done! and changed in both new entrypoint methods